pkg/cnab/provider/action.go

```diff
@@ -73,6 +73,10 @@ func (d *Runtime) AddRelocation(args ActionArguments) action.OperationConfigFunc
 			op.Files["/cnab/app/relocation-mapping.json"] = string(b)
 			var reloMap relocation.ImageRelocationMap
 			err = json.Unmarshal(b, &reloMap)
+			// If the invocation image is present in the relocation mapping, we need
+			// to update the operation and set the new iamge reference. Unfortunately,
```

Comment:
`s/iamge/image` then good to go!  Thanks for the comment/description -- helped me :)